### ISSUE TITLE
Need to change the client_id in the public/index.js file also 

### ISSUE THREAD

**Issue by garthsun** [sohamkamani/node-oauth-example#3]
The example did not work until I found out that the public/index.js also needs to be changed w.r.t. the client_id.

**Comment by vivekomgala** [comment]
Yup. You are right. I changed the clientid in public/index.html but still on Welcome page its supossed to show Welcome message which it isnt able to. Are you able to
